apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.js

```diff
@@ -77,14 +77,33 @@ router.get(
         res.locals.assessment_instance.id,
         false,
       );
-
+      const fingerprintNumbers = {};
```

Comment:
I'd suggest using a [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) here, rather than an object. The MDN page in the previous link contains a good summary of the differences between them, but in this case a Map will be more optimized for the sort of insert/query behavior that we using.

Reply:
I have now converted this to use a Map instead of an object in commit 22cfe06.